togetherly/lib/models/child.dart

```diff
@@ -10,6 +10,7 @@ class Child extends Person {
     required super.familyId,
     required super.name,
     required super.icon,
+    required super.isParent,
```

Comment:
This field was omitted because it is encoded in the fact that a Person object is either a Parent or a Child. Having a field is redundant.

Reply:
Then how will this work with the fact that almost every page has some logic to display parent or child views depending on if they are a parent or child? Can I perform that logic by referencing their class?